manager/integration/tests/common.py

```diff
@@ -2231,13 +2235,43 @@ def wait_for_backup_completion(client, volume_name, snapshot_name,
     return v
 
 
-def wait_for_backup_to_start(client, volume_name, snapshot_name,
+def wait_for_recurring_backup_completion(client, core_api,
```

Comment:
This function is waiting for the first backup completion. Also, if the volume is currently attaching to a running pod, the pod phase will be `running` while the backup is happening. 
Should we name it `wait_for_first_autoattach_recurring_backup_completion` to avoid miss-using this function in the future?

Reply:
You are right the function name should speak more for itself. The `wait_for_first_autoattach_recurring_backup_completion` is a little hard to read imo, how about `wait_pod_auto_attach_after_first_backup_completion`?

Reply:
Yeah, good idea!